docker/docker-compose.yml

```diff
@@ -83,18 +84,19 @@ services:
       - '6379'
   php:
     <<: *php
-    image: ${PHP_IMAGE:-humanmade/altis-local-server-php:3.0.0}
+    image: ${PHP_IMAGE:-humanmade/altis-local-server-php:3.1.0}
```

Comment:
Need to update this for the `start` command too: https://github.com/humanmade/altis-local-server/blob/master/inc/composer/class-command.php#L103

Reply:
Thanks, Rob! Updated 👍